settings.gradle

```diff
@@ -21,3 +22,7 @@ include 'samples:websocket'
 include 'spring-session'
 include 'spring-session-data-gemfire'
 include 'spring-session-data-redis'
+include 'spring-session-jdbc'
+include 'samples:httpsession-jdbc-xml'
+findProject(':samples:httpsession-jdbc-xml')?.name = 'httpsession-jdbc-xml'
```

Comment:
Certainly not, it seems like IntelliJ's Gradle plugin's messing with Gradle conf.
I remember fixing something similar in ```settings.gradle``` prior to pushing the initial commit but it slipped in the updated commit.